docs/src/api/class-android.md

```diff
@@ -87,6 +87,11 @@ Returns the list of detected Android devices.
 
 Optional port to establish ADB server connection.
 
+### option: Android.devices.autoInstallDriver
+- `autoInstallDriver` <[boolean]>
+
+Optional install latest playwright driver automatically. if false, firstly you should install android-driver.apk and android-driver-target.apk which are in bin directory of playwright-core.  Default to true.
```

Comment:
Our convention is that boolean flags default to false, let's rename this to `omitDriverInstall`